2nd-gen/packages/swc/components/divider/stories/divider.stories.ts

```diff
@@ -99,100 +132,99 @@ export const Anatomy: Story = {
- * - **Medium**: Used for dividing subsections on a page, or to separate different groupings of components such as panels, rails, etc.
- * - **Large**: Should only be used for page titles or section titles
+ * - **Small (s)**: Used to divide similar components such as table rows, action button groups, and components within a panel
+ * - **Medium (m)**: Used for dividing subsections on a page, or to separate different groupings of components such as panels, rails, etc.
+ * - **Large (l)**: Should only be used for page titles or section titles
+ *
+ * All sizes shown below for comparison.
  */
 export const Sizes: Story = {
-    render: () =>
-        html`${Divider.VALID_SIZES.map((size) => {
-            const label = sizeLabel(size);
-            return html`<div>
-                <h3>${label}</h3>
-                <swc-divider size=${size}></swc-divider>
-                <p>Content below the ${label.toLowerCase()} divider.</p>
-            </div>`;
-        })}`,
+    render: (args) => html`
+        <div>
+            <h3>Small</h3>
+            ${template({ ...args, size: 's' })}
+            <p>Content below the small divider.</p>
+        </div>
+        <div>
+            <h3>Medium</h3>
+            ${template({ ...args, size: 'm' })}
+            <p>Content below the medium divider.</p>
+        </div>
+        <div>
+            <h3>Large</h3>
+            ${template({ ...args, size: 'l' })}
+            <p>Content below the large divider.</p>
+        </div>
+    `,
     parameters: {
         flexLayout: true,
         styles: {
             'flex-direction': 'row',
             gap: '16px',
         },
+        'section-order': 1,
     },
     tags: ['options'],
 };
 
 /**
- * The default horizontal divider is used to separate content stacked vertically. To separate horizontal content, use the `vertical` attribute.
+ * The default horizontal divider is used to separate content stacked vertically. To separate
+ * horizontal content, use the `vertical` attribute.
  *
- * When a vertical divider is used inside of a flex container, use `align-self: stretch; height: auto;` on the divider.
+ * **Note**: When a vertical divider is used inside a flex container, apply `align-self: stretch; height: auto;` styling to the divider element.
+ *
+ * All sizes shown below for comparison.
  */
 export const Vertical: Story = {
-    args: {
-        vertical: true,
-    },
-    render: () =>
-        html`${Divider.VALID_SIZES.map((size) => {
-            const label = sizeLabel(size);
-            return html`
-                <span>${label}</span>
-                <swc-divider
-                    vertical
-                    size=${size}
-                    style="align-self: stretch; height: auto;"
-                ></swc-divider>
-            `;
-        })}`,
+    render: (args) => html`
+        <div class="story-vertical-divider-container">
```

Comment:
What's this for?

Reply:
something i was playing with the vertical line should be between content left and right and currently its in a column